apps/tasks/lib/components/task_list_tile.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_widget/static_progress_indicator.dart';
+
+/// A ListTile to Display a StaticProgressIndicator and Status
+class TaskListTile extends StatefulWidget{
+  const TaskListTile({
+    super.key,
+    required this.statusColor,
+    required this.progress,
+    required this.title,
+    required this.subTitle,
+    this.isChecked = false
+  });
+
+  /// The [Color] of [Checkbox] and [StaticProgressIndicator]
+  final Color statusColor;
+
+  /// The default value of status [Checkbox]
+  final bool? isChecked;
+
+  /// Progress as a Percentage between 0.0 to 1.0
+  final double progress;
+
+  /// The Title for [ListTile]
+  final String title;
+
+  /// The SubTitle for [ListTile]
+  final String subTitle;
```

Comment:
all these information are included in the Tasks Object that will be passed to this widget, therefore we should use a attribute `Map<String,dynamic> patient` instead of seperatly every information given by the patient object

Reply:
I used mock data classes which reduces parsing afford.
Maybe we should make progress in #168